### DISCUSSION TITLE
Parsing time values from Wikibase to datetime object

### DISCUSSION THREAD

**Discussion by dpriskorn** [LeMyst/WikibaseIntegrator#416] [Ideas]
The time value is "+2007-03-31T00:00:00Z"
I'm thinking it is a common use case to want this parsed into a datetime object and maybe we want WBI to support that?

**Comment by LeMyst** [comment]
Added in https://github.com/LeMyst/WikibaseIntegrator/pull/542

datetime can't be used with year < 1 :(

And i'm reluctant to add another dependency in WBI
